Makefile

```diff
@@ -306,3 +306,10 @@ deploy-dev:
 
 .PHONY: deploy-dev
 
+###############################################################################
+##                                  Docs                                     ##
+###############################################################################
+
+docs:
+	cd docs && mintlify dev
+.PHONY: docs
```

Comment:
```suggestion
docs:
	@cd docs && mintlify dev
	
.PHONY: docs
```

Reply:
Maybe we could check if you have mintlify installed too or point to where to dl

Reply:
[make docs command checks for install](https://github.com/skip-mev/slinky/pull/604/commits/d8fa59c8b1901bbcdb95dbce55174aaf5a0fb9f3)